samples/build/build_buildkit_cr.yaml

```diff
@@ -4,11 +4,16 @@ kind: Build
 metadata:
   name: buildkit-build
   annotations:
-    build.build.dev/build-run-deletion: "true"
+    build.shipwright.io/build-run-deletion: "true"
```

Comment:
note: following comment is for all changes in the strategies in general.


We need a strategy using the build and push in the strategy, so that users can have a reference for this if they want to opt-out from SHP manage-push. Where would that strategy be?

Reply:
Good question. We would still have Buildpacks that only supports pushing to container registry. If that's not enough, then we can also decide to keep a duplicate of one or two more strategies with the push performed by the strategy itself.

Reply:
Done with BuildAh.